locale/exchangecalendar/en-US/oofSettings.dtd

```diff
@@ -1,13 +1,13 @@
-<!ENTITY title.oofsettings "Out of Office settings">
+<!ENTITY title.oofsettings "Out of office settings">
```

Comment:
Outlook uses "Out of Office" in menu commands, but that seems to be the UI style for commands (the same way you pointed out elsewhere that Thunderbird uses title case for menu commands).  In the online documentation (such as [Send automatic out of office replies from Outlook](https://support.office.com/en-gb/article/Send-automatic-out-of-office-replies-from-Outlook-9742f476-5348-4f9f-997f-5e208513bd67), Microsoft tends to use "out of office" or "out-of-office" to refer to the generic functionality.

Given this, should I keep it as "out of office" or change it back to "Out of Office"?